tests/integration/targets/azure_rm_appgateway/tasks/main.yml

```
@@ -8,7 +8,9 @@
   ansible.builtin.set_fact:
     rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
     cert1_file: "cert1.txt"
+    cert1_password: "your-password"
```

Please delete the fourth line
```suggestion
    cert1_password: "{{ lookup('ansible.builtin.password', '/dev/null', chars=['ascii_letters', 'digits', 'punctuation'], length=12) }}"
```